stages/org.osbuild.oscap.autotailor

```diff
@@ -49,6 +52,20 @@ SCHEMA = """
         "type": "array",
         "items": { "type": "string" },
         "description": "The rules to deselect from the base OpenSCAP profile"
+      },
+      "overrides": {
```

Comment:
Could someone also want to completely remove a variable, not just override it? I don't know if such a thing is possible in the OpenSCAP profile or not, so ignore this if it makes no sense :)

Reply:
There is no way to remove an XCCDF variable, one could only change the value in tailoring.

Reply:
We can however `unselect` a rule, meaning that the scan won't check for compliance on that rule. That's already supported :)